incubator/hnc/go.mod

```diff
@@ -14,4 +14,5 @@ require (
 	k8s.io/apimachinery v0.0.0-20190404173353-6a84e37a896d
 	k8s.io/client-go v11.0.1-0.20190409021438-1a26190bd76a+incompatible
 	sigs.k8s.io/controller-runtime v0.2.2
+	sigs.k8s.io/controller-tools v0.2.0-beta.4 // indirect
```

Comment:
Please revert the changes to `go.mod` and `go.sum`, these just happened because you called `go get controller-tools` while in this directory but we don't actually want a dependency on them.

Reply:
Oh, my, glob. That took some git magic. I learned a lot! Thanks!